pkg/network/encoding/format.go

```diff
@@ -295,14 +297,18 @@ func unsafeStringSlice(key string) []byte {
 //				model.ProtocolType_protocolHTTP2,
 //			},
 //		}
-func formatProtocol(protocol network.ProtocolType) *model.ProtocolStack {
+func formatProtocol(protocol network.ProtocolType, staticTags uint64) *model.ProtocolStack {
+	stack := make([]model.ProtocolType, 0)
+	if staticTags&tlsTagsMask > 0 {
```

Comment:
i think we should create a function `IsTLSTag` next to `GetStaticTag`
and check it there. furthermore, add a comment next to the definitions of the TLS tags